gclient/src/utils.rs

```diff
@@ -33,19 +30,11 @@ pub fn wat2wasm(wat: impl AsRef<str>) -> Vec<u8> {
         .to_vec()
 }
 
-/// Return the time elapsed since the Unix epoch in microseconds.
-pub fn now_in_micros() -> u128 {
-    SystemTime::now()
-        .duration_since(UNIX_EPOCH)
-        .expect("Infallible")
-        .as_micros()
-}
-
 /// Return the time elapsed since the Unix epoch in microseconds converted to
 /// the 16-bytes array.
-#[deprecated = "Use `now_in_micros().to_le_bytes()` instead"]
+#[deprecated = "Use `gear_utils::now_micros().to_le_bytes()` instead"]
```

Comment:
this fn may already be deleted

Reply:
but I re-export it from the gclient. Or is it not needed there?

Reply:
Seems not. Or reexport utils anytime you need such

Reply:
done